Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Services/LocatorStorageService.cs

```diff
@@ -13,24 +13,23 @@ public class LocatorStorageService
     //TODO: Impl factory
     public T Get<T>(string elementName)
     {
+        if (elementName.Equals("")) throw new Exception("element name can not be empty. Please correct test step");
```

Comment:
It's not the best way to check string emptiness like this.
1. Better use `string.IsNullOrEmpty(elementName)`
2. There is no need to create new string with empty value `""`, you can use `string.Empty` for this case

Reply:
Changed to string.IsNullOrEmpty(elementName)

Thanks!